src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -2695,15 +2695,15 @@ public static void Main()
 ";
 
             ParseAndValidate(text, TestOptions.Regular,
-                // (7,21): error CS1526: A new expression requires an argument list or (), [], or {} after type
+                // (7,21): error CS1031: Type expected
```

Comment:
This seems wrong, especially in the face of target-typed `new`. A type is not necessarily expected here.

Reply:
In parsers it's very common to just state what the expectation is for the common case when things are missing.  It's not like the otehr error message is correct either.  it's acting as if we have a type (we don't, we have a missing identifier), and then saying what is missing coming after that missing identifier.

Reply:
Sure. But I would prefer that we introduce a message that says something like "Type or parens expected".

Reply:
i'm not opposed to that.  it's just generally not something we commonly do.  With missing stuff, we commonly just specify the 95% case we expect.  It helps keeps teh errors relevant and not overloaded with tons of information.   

I'd honestly prefer that if we do this, we do it through a "consistifying pass" that did this for all messages instead of doing it in an adhoc fashion.

Reply:
I'm not opposed to a consistifying pass. But I also think we should fix them where they're obvious

Reply:
It's not obvious to me that anything is wrong with this will message. I.e. I don't think it's the goal of a parse error to enumerate all potential legal follow-up tokens/nodes.

Reply:
@CyrusNajmabadi, my opinion on this hasn't changed. This is a worse error message, and I do not want to take the change with it.